src/Core/Checkout/Document/DocumentEntity.php

```diff
@@ -47,7 +47,7 @@ class DocumentEntity extends Entity
     /**
      * @var string
      *
-     * @deprecated tag:v6.7.0 - Will be natively typed
+     * @deprecated tag:v6.7.0 - Will be removed
      */
     protected $fileType;
```

Comment:
If the property is going to be removed, deprecate the getter and setter as well.

Reply:
Also if we remove that property from the DAL entity, why not deprecating the field in the definition as well (and ideally also add an If with the major flag there) + Major DB migration to remove it.